VCFv4.3.tex

```diff
@@ -564,23 +583,28 @@ \section{INFO keys used for structural variants}
 \end{verbatim}
 \normalsize
 \end{samepage}
-This key can be derived from the REF/ALT fields but is useful for filtering.
-The reserved values must be used for the types listed below:
+SVTYPE is a primitive basic data type and must be specified for all structural variants.
+Its value can usually be deduced from the REF and ALT fields, however despite this redundancy it is quite useful for filtering.
+For each VCF record, SVTYPE must be assigned exactly one of the following values:
 \begin{itemize}
-  \item DEL: Deletion relative to the reference
-  \item INS: Insertion of novel sequence relative to the reference
-  \item DUP: Region of elevated copy number relative to the reference
-  \item INV: Inversion of reference sequence
-  \item CNV: Copy number variable region (may be both deletion and duplication)
-  \item BND: Breakend
+\item DEL: Deletion relative to the reference
+\item INS: Insertion relative to the reference
+\item DUP: Region of elevated copy number relative to the reference
+\item INV: Inversion of reference sequence
+\item CNV: Copy number variable region (multiallelic)
```

Comment:
Maybe this is a silly question, but what does "multiallelic" mean here? The same as "may be both deletion and duplication" to say that both copynumber 0 and greater than 1 is allowed? "multiallelic" doesn't look clear to me.

Reply:
Multiallelic could mean either the presence of both a simple deletion and duplication allele for the segment, or that there are duplication alleles with differing numbers of copies. For example, one allele might have a single tandem duplication of the segment (and therefore two copies of the reference segment) and another allele might have three or more copies. See for example https://www.ncbi.nlm.nih.gov/pubmed/25621458.

Reply:
I mostly agree with @cwhelan's definition, although this needs to be reflected in the spec itself (not just this discussion)

Reply:
Extracted as a separate issue: https://github.com/samtools/hts-specs/issues/548